core/Azure.Mcp.Core/src/Areas/Server/Commands/ServiceInfoCommand.cs

```diff
@@ -0,0 +1,66 @@
+﻿// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT License.
+
+using Azure.Mcp.Core.Commands;
+using Azure.Mcp.Core.Configuration;
+using Microsoft.Extensions.Logging;
+using Microsoft.Extensions.Options;
+
+namespace Azure.Mcp.Core.Areas.Server.Commands;
+
+/// <summary>
+/// Command that provides basic server information.
+/// </summary>
+[HiddenCommand]
+public class ServiceInfoCommand(IOptions<AzureMcpServerConfiguration> serverOptions, ILogger<ServiceInfoCommand> logger) : BaseCommand<EmptyOptions>
+{
+    private readonly IOptions<AzureMcpServerConfiguration> _serverOptions = serverOptions;
+    private readonly ILogger<ServiceInfoCommand> _logger = logger;
+
+    public override string Id => "add0f6fe-258c-45c4-af74-0c165d4913cb";
```

Comment:
where did this GUID value come from? We usually avoid hard-coding GUID values in code.

Reply:
This GUID came from running New-Guid.  There was no way to map a coded \*Command to one executed by our MCP server because names and area names can change.  The Guid is Immutable.

Reply:
@KarishmaGhiya is the suggestion to move the guid to a central file with a list of Guid constants and then reference the named constant here?

Reply:
Can you open an issue with this suggestion?
1. How the file format will look.
2. Where it will live
3. What validations we need to add

Reply:
The goal should be for the server -> tools -> core dependency graph to be unidirectional, with the server -> tools dependency being just a wildcard `server.csproj` -> `tools/Azure.*` project reference.  Having even a central Program.cs where we have to register all the tools is a code smell I'm trying to remove through code gen.

Reply:
@hallipr Will code gen not affect hard-coded constants? Or will the code gen only touch "tools/*"? I was suggesting something that would be compatible with code-gen. 
But I don't want to block this PR on this discussion